src/SprykerEco/Client/Payone/PayoneClientInterface.php

```diff
@@ -163,4 +165,17 @@ public function sendAddressCheckRequest(QuoteTransfer $quoteTransfer): AddressCh
      * @return \Generated\Shared\Transfer\ConsumerScoreResponseTransfer
      */
     public function sendConsumerScoreRequest(QuoteTransfer $quoteTransfer): ConsumerScoreResponseTransfer;
+
+    /**
+     * Specification:
+     * - Sends request to Payone to start session.
```

Comment:
* Sends a request to Payone to start Klarna session.

Reply:
++